.eslintrc.json

```diff
@@ -26,21 +26,14 @@
         }
     },
     "rules": {
-        "indent": [
-            "error",
-            4,
-            {
-                "SwitchCase": 1
-            }
-        ],
         "linebreak-style": [
             "error",
             "unix"
         ],
-        "quotes": [
-            "error",
-            "single"
-        ],
+        // "quotes": [
+        //     "error",
+        //     "single"
+        // ],
```

Comment:
We can just delete this if it doesn't cause issues.

Reply:
Oops